aggregator/src/test/java/org/jenkinsci/plugins/workflow/WorkflowRunTest.java

```diff
@@ -232,4 +230,34 @@ public void failedToStartRun() throws Exception {
         r.assertBuildStatusSuccess(r.waitForCompletion(b));
     }
 
+    @Issue("JENKINS-29571")
+    @Test public void buildRecordAfterRename() throws Exception {
```

Comment:
@stephenc one additional note, @jglick can do any coding style here. But because of workflow changes enforced to all plugins and core, it must be readable for everybody. From reading code from all plugins and core i see only two difficulties for me :crying_cat_face: :
- random mess of spaces (or at all no spaces) by KK that followed by other contributors
- this annotation inlining by @jglick

Plus i don't understand what is the problem to add checkstyle to absolutely new plugin and exclude future coding style mess.